src/App.tsx

```diff
@@ -1,40 +1,49 @@
-import React from 'react';
+import { FC, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
 
-export const tabs = [
+export const tabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App: React.FC = () => {
+type Tab = {
+  id: string;
+  title: string;
+  content: string;
+};
+
+export const App: FC = () => {
+  const [selectedTabId, setselectedTabId] = useState('tab-1');
+
+  const handleChangeTab = (event: React.MouseEvent, id: string) => {
+    event.preventDefault();
```

Comment:
do you really need it?

Reply:
what do you mean? is it better to keep the handler function in the attribute?